packages/pharos-site/static/guidelines/button.docs.mdx

```diff
@@ -306,8 +306,8 @@ import ellipsesExample from '@images/components/button/buttons_ellipses-example.
   <PageSection title="Buttons on dark background" subSectionLevel={1}>
     <p>
       When a button is used on a darker background (e.g., black or marble-gray-20), use the
-      on-background attribute. The four variants described above can all be used on dark backgrounds
-      and follow the same guidelines.
+      is-on-background attribute. The four variants described above can all be used on dark
```

Comment:
do we want to hold of on changes to the `pharos-site` until the release is complete and the site is updated to 14?

Reply:
Nice catch! Definitely want to remove that 👍🏻